Heimdallr/OAuthAccessToken.swift

```diff
@@ -24,11 +27,12 @@ public class OAuthAccessToken: NSObject {
     ///
     /// - returns: A new access token initialized with access token, type,
     ///     expiration date and refresh token.
-    public init(accessToken: String, tokenType: String, expiresAt: Date? = nil, refreshToken: String? = nil) {
+    public init(accessToken: String, tokenType: String? = nil, expiresAt: Date? = nil, refreshToken: String? = nil, payload: [String: Any]? = nil) {
```

Comment:
???: Should Bearer be the default value?

Reply:
Why are `tokenType` and `refreshToken` optional now? Is there a scenario where that would make sense?